demos/stateful-ha-blog-post/README.md

```diff
@@ -0,0 +1,415 @@
+*   If you run a multi-replica StatefulSet (and your application’s data is already replicated at the application layer), applying a HighAvailabilityApplication resource reduces the amount of time your data is not fully replicated in the event of a node or zone failure.
+
+
+## Configuring the HighAvailabilityApplication Resource
+
+Use the following specification fields to configure settings that apply to your application.
+
+
+### .spec.resourceSelection
+
+You can select the application to be managed by a HighAvailabilityApplication resource. This is done through the `.spec.resourceSelection` field. The HighAvailabilityApplication resource should be installed in the same namespace as the referenced application. Currently only `resourceKind: StatefulSet` is supported.
+
+
+### .spec.policy.storageSettings.requireRegionalStorage
+
+This setting validates the `HighAvailabilityApplication` resource to ensure your StatefulSet application is using a regional `StorageClass`. Currently, the Stateful HA operator supports only the use of the HighAvailabilityApplication resource with [Regional Persistent Disk](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/regional-pd) storage.
+
+
+### .spec.policy.failoverSettings
+
+In Kubernetes, a Pod is not deleted automatically when a [node becomes unreachable](https://kubernetes.io/docs/tasks/run-application/force-delete-stateful-set-pod/#force-deletion). When running StatefulSet Pods in a Kubernetes cluster today, it could take 8 minutes or more for your application pods to be deleted and rescheduled to a new node, depending on your cluster’s [Node Repair Strategy](https://cloud.google.com/anthos/clusters/docs/on-prem/latest/how-to/node-auto-repair). A HighAvailabilityApplication resource can be configured to reschedule your Pods after a custom timeout to limit the period of time that a Pod is unavailable.
+
+Setting the `.forceDeleteStrategy: AfterNodeUnreachable` strategy allows the Stateful HA operator to reschedule your StatefulSet pods after a timeout, specified by the `afterNodeUnreachableSeconds` field. Note that it takes ~40 seconds for a node to be marked as unreachable, so when a node first fails to succeed a health check, your application’s Pods will be unavailable for a minimum of 40 seconds. The `afterNodeUnreachableSeconds` is the amount of time the Stateful HA operator waits after a node is marked as unreachable.
+
+### Resources
+
+*   [HighAvailabilityApplication - Resource Definition](https://drive.google.com/file/d/1ix1DJlHJlRnAZA7E_6oixv9fRZ9pP8_1/view)
+*   [Demo Video](https://www.youtube.com/watch?v=yVc0Hs7Bu6w)
+
+
+## Installing the operator
+
+The operator runs as a Deployment in your cluster. To install the operator and `HighAvailabilityApplication` custom resource definition, run the following commands:
+
```

Comment:
The commands are missing?

Reply:
This was removed when converting. Updated.